wiki/troubleshooting.md

```diff
@@ -8,6 +8,9 @@ To scan and repair those files, run `DISM /Online /Cleanup-Image /RestoreHealth`
 
 For more information about running DISM check [this page](https://support.microsoft.com/en-us/help/947821/fix-windows-update-errors-by-using-the-dism-or-system-update-readiness)
 
+::: tip
```

Comment:
Making it lowercase cuz TIP is a bit aggressive imo.

```suggestion
::: tip Tip
```

Reply:
This should be called "Note" instead imo.

Reply:
All done. I'll mark this as resolved.